README_RASPBERRY_PI.md

```diff
@@ -0,0 +1,189 @@
+        s.waitForBoot{ {SinOsc.ar([400, 404])}.play }
+
+    sudo reboot # the sound should start after a few seconds
+
+Login with ssh and run `killall jackd sclang scsynth` to stop the sound.
+
+Benchmarks
+----------
+
+These are rough benchmark tests. The server should be booted and jackd running with settings: `-P75 -p1024 -n3 -r44100`
+
+Also for comparison it is important to set CPU scaling to 'performance', by running:
+
+    echo performance | sudo tee /sys/devices/system/cpu/cpu*/cpufreq/scaling_governor
+
+Start sclang or scide and run:
+
+    s.boot
+    {1000000.do{2.5.sqrt}}.bench // ~0.56 for rpi3 headless, ~0.7 for rpi3 scide, ~1.7 for rpi0 headless, ~3.8 for rpi0 scide
+    a= {Mix(50.collect{RLPF.ar(SinOsc.ar)});DC.ar(0)}.play
+    s.avgCPU // run a few times. ~12% for rpi3, ~18% for rpi2, ~79% for rpi1, ~50% for rpi0
+    a.free
+
+With the default cpu scaling (ondemand) these benchmarks perform much worse, but 'ondemand' also saves battery life so
+depending on your application, this might be the preferred mode.
+
+To set 'performance' scaling mode permanently see the "Gotcha..." section of [this StackExchange
+post](https://raspberrypi.stackexchange.com/questions/9034/how-to-change-the-default-governor#9048).
+
+Notes and Troubleshooting
+-------------------------
+
+This applies to both GUI and GUI-less builds above:
+
+* An easy way to burn the zip file (no need to unpack) to an SD card is to use [etcher](http://etcher.io).
+* The internal soundcard volume is by default set low (40). Type `alsamixer` in terminal and adjust the pcm volume to
+  85 with the arrow keys. Press escape to exit.
+* The audio quality of rpi's built-in sound is terrible. Dithering helps a bit so add `-zs` to the jackd command if
+  you are using the built-in sound.
+* If building with `-j 3` stops or returns an error the compiler might just have run out of memory. Try to reboot and
+  run the make command again without `-j 3` or decrease the gpu memory in raspi-config under advanced (set it to 16).
+* If you get `WARNING: REMOTE HOST IDENTIFICATION HAS CHANGED!` when trying to ssh in, type `ssh-keygen -R
+  raspberrypi` to reset.
+* For lower latency, set a lower blocksize for jackd. Try, for example `-p512` or `-p128`. Tune downwards until you
+  get dropouts and xruns (also watch cpu%).
+* To avoid SD card corruption one should always shut down the system properly and not just pull out the power. When
+  running headless you can either ssh in and type `sudo halt -p`, use a GPIO pin with a button and Python script, or
+  set up an OSC command from within SC that turns off the RPi. See
+  [here](https://github.com/blacksound/VTM/wiki/Raspberry-Pi-Instructions#shutdown-for-raspberry-pi).
+* For the older Raspbian Jessie system use a [previous](https://github.com/supercollider/supercollider.github.io/blob/1f578b5fa71e1acae0ce40d14bc0ef116062093d/development/building-raspberrypi.md)
+  version of these instructions.
```

Comment:
is the newline needed here?

Reply:
yes, with my editor's column limit (as you can see from the rest of the file)